packages/amplify_analytics_pinpoint/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 0.1.3 (2021-04-21)
+
```

Comment:
*nit:* Should we add something like, "dependency for amplify_core changed to 0.1.3"?

I would be confused if I saw a change log entry with no info in it at all.

Reply:
I agree.  I'm not going to do it here, since we haven't had a chance to discuss. We have a few of these entries already, so it's not a new problem.